style.css

```diff
@@ -128,3 +127,293 @@
 .icons:hover {
   background-color: #fff5e1;
 }
+
+.works {
+  background-color: #fff;
+  width: 100%;
+}
+
+.frame {
+  width: 100%;
+}
+
+#heading {
+  height: 107px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: 120px;
+  margin-left: 6.4%;
+  margin-right: 6.4%;
+  text-align: center;
+}
+
+#yoga-img {
+  width: 87.2%;
+  margin-left: 6.4%;
+  margin-right: 6.4%;
+}
+
+#heading3 {
+  height: 36px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  margin-top: 24px;
+  margin-left: 6.4%;
+  margin-right: 6.4%;
+}
+
+.framepara {
+  height: 147px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-left: 6.4%;
+  margin-right: 6.4%;
+}
+
+#tagsul {
+  display: flex;
+  list-style: none;
+  width: 297px;
+  height: 40px;
+  gap: 12px;
+  align-items: center;
+  margin-top: 12px;
+  margin-left: 6.4%;
+  margin-right: 6.4%;
+}
+
+.tags1 {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  box-sizing: border-box;
+  padding: 10px 12px;
+  border: 1px solid #8993a4;
+  align-items: center;
+}
+
+#project-button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border-style: none;
+  margin-top: 12px;
+  margin-left: 6.4%;
+}
+
+.cards {
+  display: grid;
+  padding-left: 6.4%;
+  padding-right: 6.4%;
+  padding-top: 3.76%;
+  padding-bottom: 120px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./Work-section-img/backgroundli.png);
+  height: 386px;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  margin-bottom: 3.76%;
+  width: 100%;
+}
```

Comment:
Kindly adjust the project cards to centralize on the page as given on the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/219849140-8f4fadbc-0eab-4247-93b7-d059a4d3a49d.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/219849212-81e38d29-169f-4d06-88e2-c3c0ea9e9055.png)

Reply:
Do i need to change inside content? or whole the container needs to be centralized.

Reply:
Hi @vikas924,

From the figma design, we should have equal width on both sides of the project card.